servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/gc/AbstractRestGCTest.java

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.jaxrs.gc;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.error.BaseNessieClientServerException;
+import org.projectnessie.model.Branch;
+import org.projectnessie.model.Content;
+
+public abstract class AbstractRestGCTest extends AbstractRestGC {
+
+  @Test
+  public void testSingleRefMultiTable() throws BaseNessieClientServerException {
+    String prefix = "test1";
```

Comment:
Those should really be the name of the test method - not a number (for all tests)